src/main/java/echoServer/http/Router.java

```diff
@@ -0,0 +1,16 @@
+package echoServer.HTTP;
+
+import echoServer.http.StatusCode;
+
+public class Router {
+
+    public static echoServer.http.StatusCode router(HTTPRequest request) {
```

Comment:
If you've imported `StatusCode` above then you don't need the fully qualified name of the class here in the return type.

Reply:
Also, I've noticed that you've chosen to use static methods in both the `Router` and the `RequestParser`. While that works for right now, I would really encourage you to not just treat your classes as bags of functions but actually think about what attributes should be encapsulated in the class. Remember how _Philosophy of Software Design_ recommended building your classes to have deep rather than broad interfaces.